packages/trader/src/Modules/MT5/Containers/mt5-dashboard-container.jsx

```diff
@@ -40,7 +41,7 @@ const MT5DashboardContainer = () => (
                     </a>
                     <a
                         className='mt5-dashboard__download-center-options--desktop-link--top'
-                        href='https://trade.mql5.com/trade?servers=Binary.com-Server&trade_server=Binary.com-Server'
+                        href={getMT5WebTerminalLink({ category: 'real', loginid: '' })}
```

Comment:
in the `getMT5WebTerminalLink` function, shouldn't we remove the `&login=` at the end if login is undefined and not pass it in here?